include/QuadratureLfo.h

```diff
@@ -30,51 +30,68 @@
 class QuadratureLfo
 {
 public:
-	QuadratureLfo( int sampleRate );
+	QuadratureLfo( int sampleRate ) :
+		m_frequency(0),
+		m_phase(0),
+		m_offset(D_PI / 2)
+	{
+		setSampleRate(sampleRate);
+	}
+
 	~QuadratureLfo()
 	{
 	}
 
 	inline void setFrequency( double frequency )
 	{
-		if( frequency < 0 || frequency > ( m_samplerate / 2.0 ) || frequency == m_frequency )
+		if( frequency < 0 || frequency > m_samplerate / 2.0 || frequency == m_frequency )
 		{
 			return;
 		}
 		m_frequency = frequency;
 		m_increment = m_frequency * m_twoPiOverSr;
 
-		if( m_phase >= F_2PI )
+		if( m_phase >= D_2PI )
 		{
-			m_phase -= F_2PI;
+			m_phase -= D_2PI;
 		}
 	}
 
 
-
-
 	inline void restart()
 	{
 		m_phase = 0;
 	}
 
 
-
-
 	inline void setSampleRate ( int samplerate )
 	{
 		m_samplerate = samplerate;
 		m_twoPiOverSr = F_2PI / samplerate;
 		m_increment = m_frequency * m_twoPiOverSr;
 	}
 
-	void tick( float *s, float *c );
+
+	inline void setOffset( double offsetVal )
+	{
+		m_offset = offsetVal;
+	}
+
+
+	void tick( float *l, float *r )
+	{
+		*l = sinf( m_phase );
```

Comment:
Out of scope of this pull request, but it is not good practice to use standard library trigonometric functions as they are slow in all the common implementations and have no bound time guarantees, it would be normal to use a fast approximation or lookup tables.